src/guides/v2.3/extension-dev-guide/staging.md

```diff
@@ -6,6 +6,9 @@ ee_only: true
 
 Staging functionality enables you to create scheduled campaigns, manage campaigns on dashboard and manipulate each future store object update. Moreover, you can preview a [storefront](https://glossary.magento.com/storefront) during the future campaign.
 
+{:.bs-callout-info}
+Information on how the Staging functionality works and how to use the various features, can be found on the [Content Staging Guide]({{ site.baseurl }}/user-guide/cms/content-staging.html).
```

Comment:
Hi @BarnyShergold. Could you double-check that the link goes to an existing page? For some reason, I can't find it. 

Thanks

Reply:
Found the problem - right page, wrong domain. It's odd, I'll swear it was working :)